tests/XPath2.Tests/XPath2Tests.cs

```diff
@@ -105,5 +105,23 @@ public void XPath2_substring()
 
             Assert.Equal("", result);
         }
+
+        [Fact]
+        public void XPath2_number()
+        {
+            var xml = new XmlDocument
+            {
+                InnerXml = @"
+                    <root num='134'>
+                        <numeric>456</numeric>
```

Comment:
Use a `.` here for testing purposes.
```suggestion
                        <numeric>456.78</numeric>
```

Reply:
I will update the test